subprojects/docs/src/docs/release/notes.md

```diff
@@ -49,6 +49,22 @@ Android Studio for instance will use this API to request just the dependencies f
 
 For more information see the [documentation](javadoc/org/gradle/tooling/provider/model/ParametrizedToolingModelBuilder.html) of the new API.
 
+### Security upgrade of third-party dependencies
+
+This version includes several upgrade of third-party dependency packages to fix the following security issues:
+
+- CVE-2017-7525 (critical)
```

Comment:
We should probably add links to these issue. I wouldn't expect user to know what they are about. A link would help users to better understand the vulnerability that has been fixed.

Reply:
Good advice. Not all issues are public (looks like SONATYPE-XXs are private), so I added links to all CVE issues.